web/vtadmin/src/components/routes/workflow/Workflow.tsx

```diff
@@ -54,28 +64,56 @@ export const Workflow = () => {
                 </NavCrumbs>
 
                 <WorkspaceTitle className="font-mono">{name}</WorkspaceTitle>
+                {isReshard && (
+                    <div className={style.headingMetaContainer}>
+                        <div className={style.headingMeta}>
+                            <span>
+                                {data?.workflow?.source?.shards?.length! > 1 ? 'Source Shards: ' : 'Source Shard: '}
+                                {data?.workflow?.source?.shards?.map((shard) => (
+                                    <code key={`${keyspace}/${shard}`}>
+                                        <ShardLink clusterID={clusterID} keyspace={keyspace} shard={shard}>
+                                            {`${keyspace}/${shard} `}
```

Comment:
Do we want the trailing whitespace here for some reason?

Reply:
This wasn't a good way to display different shards. Used `margin-right` instead of trailing space.